docusaurus/docusaurus.config.js

```diff
@@ -505,33 +505,49 @@ const config = {
             position: 'right',
             items: [
               {
-                label: 'Blog',
-                href: 'https://openziti.io',
+                type: 'html',
+                value: '<div class="text-divider"><p>Socials</p></div>'
+              },
+              {
+                type: 'html',
+                value: '<a href="https://www.youtube.com/OpenZiti" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/yt.svg"/>YouTube</span></a>'
+              },
+              {
+                type: 'html',
+                value: '<a href="https://twitter.com/OpenZiti" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/twit.svg"/>Twitter</span></a>'
               },
               {
-                label: 'YouTube',
-                href: 'https://www.youtube.com/openziti',
+                type: 'html',
+                value: '<a href="https://reddit.com/r/openziti" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="https://www.redditinc.com/assets/images/site/reddit-logo.png"/>Reddit</span></a>'
               },
               {
-                label: 'Forum',
-                href: 'https://openziti.discourse.group',
+                type: 'html',
+                value: '<span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/ziggy.png"/><a href="https://twitter.com/OpenZiggy" target="_blank">OpenZiggy\'s Twitter</span></a>'
               },
               {
-                label: 'Twitter',
-                href: 'https://twitter.com/openziti',
+                type: 'html',
+                value: '<div class="text-divider"><p>Other</p></div>'
               },
               {
-                label: 'Ziggy\'s Twitter',
-                href: 'https://twitter.com/openziggy',
+                type: 'html',
+                value: '<span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/oz-test-kitchen.png"/><a href="https://github.com/openziti-test-kitchen" target="_blank">Open Ziti Test Kitchen</span></a>'
               },
               {
-                label: 'Reddit',
-                href: 'https://reddit.com/r/openziti',
-              }
+                type: 'html',
+                value: '<a href="https://zeds.openziti.org" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/zeds.png"/>ZEDS</span></a>'
+              },
+              {
+                type: 'html',
+                value: '<a href="https://netfoundry.io/pricing/" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/nf.svg"/>CloudZiti</span></a>'
+              },
+              {
+                type: 'html',
+                value: '<a href="https://www.linkedin.com/company/netfoundry/" target="_blank"><span id="navbarDropdownItem"><img id="navbarDropdownImage" src="img/li.svg"/>NetFoundry</span></a>'
+              },
```

Comment:
Those two existing socials (Discourse, GitHub) are probably still the two most important, but part of me wants to present all of them the same way. Here's how it looks on mobile. 

![Screenshot_20221209-192917](https://user-images.githubusercontent.com/1434400/206818738-e9eed303-4b80-40bb-a5ce-44c0763de1e0.png)